public/js/stores/ArticleStore.js

```diff
@@ -0,0 +1,28 @@
+import { EventEmitter } from 'events'
+import dispatcher from '../dispatcher.js'
+
+class ArticleStore extends EventEmitter {
+    constructor() {
+        super()
+        this.articles = []
+        this.handleAction = this.handleAction.bind(this)
+    }
+
+    getArticles() {
+        return this.articles
+    }
+
+    handleAction(action) {
+        switch(action.type) {
+            case 'UPDATE_ARTICLES': {
+                this.articles = action.articles 
+                this.emit('update')
```

Comment:
This is cool! Should ```'UPDATE_ARTICLES'``` and ```'update'``` be named the same? Or is it fine since they aren't the same?